src/building/how-to-build-and-run.md

```diff
@@ -6,15 +6,23 @@ be hacking on `rustc`, you'll want to tweak the configuration of the compiler.
 The default configuration is oriented towards running the compiler as a user,
 not a developer.
 
+## Get the source code
+
+The very first step to work on `rustc` is to clone the repository:
+
+```text
+git clone https://github.com/rust-lang/rust.git
+cd rust
```

Comment:
Am I right that initializing the submodules is taking care of by `./x.py`, so no manual step required for that?

Reply:
Yeah, that's right.